sdk/python/kfp/cli/run.py

```diff
@@ -81,7 +82,10 @@ def submit(ctx, experiment_name, run_name, package_file, pipeline_id, pipeline_n
     experiment = client.create_experiment(experiment_name)
     run = client.run_pipeline(experiment.id, run_name, package_file, arg_dict, pipeline_id,
                               version_id=version)
-    _display_run(client, namespace, run.id, watch, output_format)
+    if timeout > 1:
```

Comment:
`1` is still a valid timeout. If use ever sets timeout to `1`, the code still needs to behavior correctly, even thought the difference is likely unnoticeable.

Reply:
You are right. Thanks for review!

Reply:
DONE